src/connector/src/sink/iceberg/mod.rs

```diff
@@ -283,7 +284,18 @@ impl IcebergSink {
                 names.push(self.config.common.table_name.to_string());
                 match &self.config.common.warehouse_path {
                     Some(warehouse_path) => {
-                        if warehouse_path.ends_with('/') {
+                        let url = Url::parse(warehouse_path);
+                        if url.is_err() {
+                            // For rest catalog, the warehouse_path could be a warehouse name.
```

Comment:
Shall we use another option for warehouse name?

The rustdoc of `warehouse_path` says "only applicable in storage catalog"

Reply:
Actually, spark uses `warehouse` key to allow users to specify warehouse name or warehouse path. However, in our system, we use warehouse.path instead. Maybe we should use a warehouse alias.